kernel-modules/probe/collector_probe.c

```diff
@@ -316,7 +314,7 @@ static __always_inline int exit_probe(long id, struct sys_exit_args* ctx) {
   }
```

Comment:
There is a `mapped_id` reference remaining.

Reply:
dang, good catch! thanks